ibm-cloud-pak-for-data/azuredeploy.json

```diff
@@ -0,0 +1,2593 @@
+			{ "name": "[variables('newStorageAccountInfra')]", "tagName": "InfraStorageAccount", "skuName": "[variables('storageSkuObject').infra.skuName]", "skuTier": "[variables('storageSkuObject').infra.skuTier]" },
+			{ "name": "[variables('newStorageAccountNodeOs')]", "tagName": "NodeStorageAccount", "skuName": "[variables('storageSkuObject').node.skuName]", "skuTier": "[variables('storageSkuObject').node.skuTier]" },
+			{ "name": "[variables('newStorageAccountNodeData')]", "tagName": "NodeStorageAccount", "skuName": "[variables('storageSkuObject').node.skuName]", "skuTier": "[variables('storageSkuObject').node.skuTier]" },
+			{ "name": "[variables('newStorageAccountPersistentVolume1')]", "tagName": "PersistentVolume1StorageAccount", "skuName": "[variables('storageSkuObject').persistent.skuName]", "skuTier": "[variables('storageSkuObject').persistent.skuTier]" }
+		],
+		"availabilitySetArray": [
+			{ "name": "masteravailabilityset", "tagName": "MasterAvailabilitySet" },
+			{ "name": "infraavailabilityset", "tagName": "InfraAvailabilitySet" },
+			{ "name": "nodeavailabilityset", "tagName": "NodeAvailabilitySet" },
+			{ "name": "bastionavailabilityset", "tagName": "BastionAvailabilitySet" }
+		],
+		"availabilitySetSku": "Aligned",
+		"platformFaultDomainCount": 2,
+		"nfsVmSize": "[parameters('masterVmSize')]",
+		"nfsInstanceCount": 1,
+		"storageSkuObject": {
+			"master": {
+				"skuName": "[variables('vmSizesMap')[parameters('masterVmSize')].storageAccountType]",
+				"skuTier": "[variables('vmSizesMap')[parameters('masterVmSize')].storageAccountTier]"
+			},
+			"infra": {
+				"skuName": "[variables('vmSizesMap')[parameters('infraVmSize')].storageAccountType]",
+				"skuTier": "[variables('vmSizesMap')[parameters('infraVmSize')].storageAccountTier]"
+			},
+			"node": {
+				"skuName": "[variables('vmSizesMap')[parameters('nodeVmSize')].storageAccountType]",
+				"skuTier": "[variables('vmSizesMap')[parameters('nodeVmSize')].storageAccountTier]"
+			},
+			"bastion": {
+				"skuName": "[variables('vmSizesMap')[parameters('bastionVmSize')].storageAccountType]",
+				"skuTier": "[variables('vmSizesMap')[parameters('bastionVmSize')].storageAccountTier]"
+			},
+			"nfs": {
+				"skuName": "[variables('vmSizesMap')[variables('nfsVmSize')].storageAccountType]",
+				"skuTier": "[variables('vmSizesMap')[variables('nfsVmSize')].storageAccountTier]"
+			},
+			"diagnostics": {
+				"skuName": "Standard_LRS",
+				"skuTier": "Standard"
+			},
+			"registry": {
+				"skuName": "Standard_LRS",
+				"skuTier": "Standard"
+			},
+			"persistent": {
+				"skuName": "Standard_LRS",
+				"skuTier": "Standard"
+			},
+			"dataStorage": {
+				"skuName": "Standard_LRS",
+				"skuTier": "Standard"
+			}
+		},
+		"storageLoopCount": 6,
+		"bastionPublicIpDnsLabel": "[concat('bastiondns', uniqueString(concat(resourceGroup().id, 'bastion')))]",
```

Comment:
let's remove the literal strings from all the uniqueString functions - this will use the same uniqueString across resources and make it easier to identify which resource belongs to which deployment/instance